lib/palletjack/tool.rb

```diff
@@ -342,29 +342,42 @@ def pallet_dir(kind, name)
       config_dir :warehouse, kind, name
     end
 
-    # Write a key box file inside a pallet
+    # :call-seq:
+    # pallet_box kind, name, box, 'key.path' => value, ...
+    # pallet_box kind, name, box { { key: { path: value, ... } } }
```

Comment:
Is it possible to move the old example, at the bottom of this comment, into this example block as well?